cmd/ike/cmd/develop_test.go

```diff
@@ -268,6 +268,8 @@ var _ = Describe("Usage of ike develop command", func() {
 					"--build", "mvn clean install",
 					"--port", "4321",
```

Comment:
We can see if we can automap the port based on DC config or Run config

Reply:
Could you open issue for it?

Reply:
https://github.com/aslakknutsen/istio-workspace/issues/35